<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_search</class>
 <widget class="QDialog" name="Dialog_search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>629</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>38</y>
     <width>1061</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Search!!!!!!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>Search mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>search by title</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>search by location</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>search by datetime</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>search by importance</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="3">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget">
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>4</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>5</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>6</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>7</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>8</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>9</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>10</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>11</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>12</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>13</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>14</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>15</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>16</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>17</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>18</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>19</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>20</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>21</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>22</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>23</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>24</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>25</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>26</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>27</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>28</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>29</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>30</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>31</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>32</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>33</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>34</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>35</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>36</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>37</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>38</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>39</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>40</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>41</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>42</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>43</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>44</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>45</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>46</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>47</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>48</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>49</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>50</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Title</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Location</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>StartDay</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>EndDay</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>StartTime</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>EndTime</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_3">
         <widget class="QWidget" name="gridLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>180</y>
            <width>221</width>
            <height>111</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>please input a title</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_5">
         <widget class="QWidget" name="gridLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>90</y>
            <width>291</width>
            <height>241</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>please input a location</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_6">
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>160</y>
            <width>302</width>
            <height>161</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="1">
            <widget class="QDateEdit" name="dateEdit"/>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="dateEdit_2"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>End</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>please select your time range</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <widget class="QWidget" name="gridLayoutWidget_8">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>110</y>
            <width>301</width>
            <height>281</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>Yes</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>important?</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>74</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
